.github/workflows/coding-statndard-check.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Coding Standard Check
```

Comment:
I think the filename has a typo? it should be `coding-standard-check.yml` right?

Reply:
Thanks.